invenio_app_ils/ui/backoffice/package.json

```diff
@@ -9,6 +9,7 @@
     "pre-build": "react-scripts build",
     "build": "npm-run-all pre-build move-js move-css",
     "test": "react-scripts test --env=jsdom",
+    "test:ci": "CI=true react-scripts test --env=jsdom",
```

Comment:
Was it for what `CI=true` again?

Reply:
To prevent jest to run in watch mode ;)

Reply:
To run jest in non watch mode!

Reply:
Right, then, what about changing it to something like:
`"test:nowatch": "nowatch=true react-scripts test --env=jsdom",`

Will be any useful to run the tests locally too? I just wouldn't "specialize it" for CI.

Reply:
Is this valid? I just saw that the CI=true is the only way to run without watch.

Reply:
Still I can change the script name!